src/gvmd.c

```diff
@@ -1813,7 +1813,8 @@ gvmd (int argc, char** argv)
           "<number>" },
         { "relay-mapper", '\0', 0, G_OPTION_ARG_FILENAME,
           &relay_mapper,
-          "Excecutable for mapping scanner hosts to relays.",
+          "Executable for mapping scanner hosts to relays."
+          " Will check $PATH for the default. Use empty string to disable.",
```

Comment:
"Will check $PATH for the default." is confusing to me.  I think it should mention
"gvm-relay-mapper" and it should be clear that $PATH is only checked if
--relay-mapper is not given.

Reply:
If changed the documentation of the option in 3f9d7651739052044f2076462b193b00ee39c4b8.